src/coord/src/coord.rs

```diff
@@ -1854,6 +1882,21 @@ where
         self.create_sink_dataflow(name.to_string(), id, sink.from, connector)
     }
 
+    fn update_catalog_view(&mut self, v: CatalogView, row: Row, diff: isize) {
```

Comment:
Would be great to document. I suspect that we will want to change this soon to support multiple updates. I suspect it is ergonomic at the moment owing to sinks not being high churn, but for anything else we'll want to bundle updates together in `SequencedCommand::Insert` messages.

Reply:
Added a comment. We can extend it when we use it for that, I think?